src/ListView/ListViewTopPanel.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PageTopPanel } from '@sailthru/stui-regions';
+import { Button } from "@sailthru/stui-elements";
+
+const Header = styled.header`
+    margin-bottom: 10px;
+    z-index: auto;
+`;
+
+const NewButton = styled.div`
```

Comment:
no need for the extra layer here

```suggestion
const StyledButton = styled(Button)`
```

Reply:
Updated...thanks!